model_compression_toolkit/core/tpc_models/default/v1_2_0/tpc_pytorch.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import operator
+
+import torch
+from torch import add, sub, mul, div, flatten, reshape, split, unsqueeze, dropout, sigmoid, tanh, chunk
+from torch.nn import Conv2d, Linear, BatchNorm2d
+from torch.nn import Dropout, Flatten, Hardtanh
+from torch.nn import ReLU, ReLU6, PReLU, SiLU, Sigmoid, Tanh, Hardswish
+from torch.nn.functional import relu, relu6, prelu, silu, hardtanh, hardswish
+
+from model_compression_toolkit.core.tpc_models.default.v1_2_0.tp_model import get_tp_model
+import model_compression_toolkit as mct
+
+tp = mct.target_platform
+
+
+def get_pytorch_tpc() -> tp.TargetPlatformCapabilities:
+    """
+    get a Pytorch TargetPlatformCapabilities object with default operation sets to layers mapping.
+    Returns: a Pytorch TargetPlatformCapabilities object for the given TargetPlatformModel.
+    """
+    default_tp_model = get_tp_model()
+    return generate_pytorch_tpc(name='default_pytorch_tpc', tp_model=default_tp_model)
+
+
+def generate_pytorch_tpc(name: str, tp_model: tp.TargetPlatformModel):
+    """
+    Generates a TargetPlatformCapabilities object with default operation sets to layers mapping.
+    Args:
+        name: Name of the TargetPlatformModel.
+        tp_model: TargetPlatformModel object.
+    Returns: a TargetPlatformCapabilities object for the given TargetPlatformModel.
+    """
+
+    pytorch_tpc = tp.TargetPlatformCapabilities(tp_model,
```

Comment:
Why you don't import it from the previous version? Same for a several...

Reply:
If you mean to import generate_pytorch_tpc and use it here, I think I prefer it this way (without importing).
I think that using functions from different versions (just because in these specific versions they're the same) is bugs prone.

Reply:
It is a problem in the case we want to use functions from version. for example: tests